rememberLastLogin isn't applicable if you have Seamless SSO enabled (#8463)
auth0/docs: articles/libraries/lock/v11/configuration.md

  rememberLastLogin: false
};
```
::: note

> Newer tenants have automatically have seamless sso enabled (https://auth0.com/docs/dashboard/guides/tenants/enable-sso-tenant). If seamless sso is enabled, the rememberLastLogin will not be relevant because if there is a session in place then the hosted login page will not be displayed at all. Using seamless sso is highly recommended because it provides a better experience for users. Users will have a seamless authentication experience: they login once and won’t have to enter credentials again when they navigate either through the applications you have built and/or third party apps. If the user is not logged in, it will redirect the user to the login screen which is expected. So, when seamless sso is enabled, users will not be prompted to confirm log in before SSO redirection.

Reviewer: There are a couple of things we should fix here:

* Seamless SSO should be capitalized appropriately throughout (as opposed to 'seamless sso')
* We could link the [text to the SSO docs](https://auth0.com/docs/dashboard/guides/tenants/enable-sso-tenant) rather than explicitly showing the URL
* Some of the wording can be reduced and made more concise. In addition, I would drop the last sentence as it's a bit redundant
* "and/or" doesn't work when you have already started the clause with "either". Just using "or" is fine here, or drop "either"
* We can put the Lock option in backticks to show that it's referring to an element of code

Can I suggest something like the following?

```suggestion
New tenants [automatically have Seamless SSO enabled](https://auth0.com/docs/dashboard/guides/tenants/enable-sso-tenant). With this enabled, the `rememberLastLogin` option will not be relevant because if there is a session in place then the hosted login page will not be displayed at all. Using Seamless SSO is highly recommended because it provides a seamless authentication experience: users log in once and won’t have to enter credentials again when they navigate either through the applications you have built, or third party apps. If the user is not logged in they will be redirected to the login screen, as expected.
```
